### ISSUE TITLE
"Check all" Checkbox in thead doesn't work with stickyHeaders

### ISSUE THREAD

**Issue by ptepper** [Mottie/tablesorter#261]
Hi,

When you place a checkbox in a TH with stickyHeaders, and then scroll down the page so that the THEAD starts floating, the 'check all checkboxes' functionality stops working. I noticed this similar issue:

https://github.com/Mottie/tablesorter/issues/83

And if you look in the last demo listed you'll see that this issue occurs there as well: http://jsfiddle.net/Mottie/abkNM/16/

I think it's making a copy of the TH when you scroll down and it sticks, so the click event might not be bound to the floating checkbox element in the TH, so I tried fixing it by adding a class to TH checkbox , and then defining the 'check all' click function using jquery ".on" so that it attaches to elements created in the future, like this:

``` javascript
$(document).on("click",".checkAll",function(){  
    if($(this).is(":checked"))
        $(".rowCheckbox").attr("checked","checked");
    else
        $(".rowCheckbox").removeAttr("checked");    
})
```

But that does not work. Any ideas?

thanks.

**Comment by ajaworska** [comment]
I'm having the same issue with stickyHeaders v2.28.4 (jquery v1.11.4, tablesorter v2.28.8).
A click on the checkbox is not  recognized.

**Comment by Mottie** [comment]
Hi @ajaworska!

The above code to handle the checkbox in the `thead` has been incorporated into the [`parser-input-select.js` file](https://github.com/Mottie/tablesorter/blob/master/js/parsers/parser-input-select.js). Please include it, or copy the code from the file - [here is a demo](http://jsfiddle.net/Mottie/y5dmhybr/).

If you continue to have problems, please modify the demo I just shared and try to replicate the issue.

**Comment by ajaworska** [comment]
Hi @Mottie,
thanx for the quick answer. I added the parser-input-select.js but it didn't solve the issue.
Then I called the demo and must say, my mentioned problem is there too. If you scroll the table so that the header sticks on the top and the first row is not visible and then click the all-select button, the button is selected but the entries in the table aren't.

**Comment by Mottie** [comment]
Ahhh you're right! I'll work on this later tonight when I get some time.

**Comment by ajaworska** [comment]
Hi @Mottie, were you able to fix this issue?
The problem seems that the click event on the checkbox isn't fired at all when the list is scrolled down... Removing it from the thead helps but isn't a nice solution.

**Comment by Mottie** [comment]
Oh! Sorry, I've had a lot of distractions recently. I'll move this higher on my priority list!

**Comment by Mottie** [comment]
This should now be all fixed in v2.28.10.
